src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLoggerOptions.cs

```diff
@@ -28,23 +28,48 @@ public class ApplicationInsightsLoggerOptions : IOptionsFormatter
         public SnapshotCollectorConfiguration SnapshotConfiguration { get; set; }
 
         /// <summary>
+        /// [Deprecated] Please use LiveMetricsAuthenticationApiKey.
```

Comment:
Use `[Obsolete]` here -- it lets you put in a string for a description and just say "Use ... instead."

Reply:
Awesome, that's what I was looking for. Thanks :)